ReadMe.md

```diff
@@ -0,0 +1,32 @@
+# Sales Reporting Tool
+This is an API system to help us record the sales that are made for our products.
+
+The solution is built with using ASP.NET Web API and currently has 2 functions, namely:
```

Comment:
I think we also we need to state the framework which is .NET 5. Also this line needs proof reading - we can cross out **with** so that it reads - **The solution is built suing ASP.NET Web API...**